### PR TITLE
Fix shadow

### PR DESCRIPTION
**Pull request by lukasoppermann**
Hey, there was a naming mistake that messes up the shadows

### CODE REVIEW

File: change/@telekom-design-tokens-ea04da6f-ec3e-4960-86b3-716443e852c7.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "fix for app bottom shadow",
```
Comment (acstll) [inline change/@telekom-design-tokens-ea04da6f-ec3e-4960-86b3-716443e852c7.json:3]: I would put some more explicit info in the comment, like `fix: rename shadow/app-bar/bottom/flat to shadow/app-bar/bottom/raised`, including the token names for reference.

File: change/@telekom-design-tokens-ea04da6f-ec3e-4960-86b3-716443e852c7.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```
[the remark this answers is not in the document]
Comment (acstll) [reply change/@telekom-design-tokens-ea04da6f-ec3e-4960-86b3-716443e852c7.json:2]: You can change this manually :)

**Review by acstll — CHANGES_REQUESTED** [review]
🙏

File: change/@telekom-design-tokens-ea04da6f-ec3e-4960-86b3-716443e852c7.json (hunk 1 above)
Comment (lukasoppermann) [reply change/@telekom-design-tokens-ea04da6f-ec3e-4960-86b3-716443e852c7.json:3]: This is not really true though, we had flat twice, so we still have flat, but now the missing raised is added.

Comment (acstll) [reply change/@telekom-design-tokens-ea04da6f-ec3e-4960-86b3-716443e852c7.json]: aaah, sorry, haha, the new comment is 💯 in any case, thank you very much!

**Review by acstll — APPROVED** [review]
🤘

**Pull request merged**